docs/developers/feature-versioning.md

```diff
@@ -0,0 +1,166 @@
+# Feature Versioning
+
+The stability levels of features (feature versioning) are independent of CRD API versions (API versioning).
```

Comment:
what does "API versioning" in brackets mean here? Did you perhaps intend to link to the "api-versioning.md"?

Reply:
Ah yes I've linked it to the doc.